requirements.txt

```diff
@@ -1 +1,3 @@
 nose==1.3.7
```

Comment:
We should remove `nose` now.

Reply:
Ok, I have also a module for python bare ready.

Reply:
As soon as this is fixed, this PR is fine for me. We could though postpone its merging until the `Python-bare` module is fixed. The only thing is that we should perhaps add some additional pytest plugins in `Python-bare`, although we don't use them now, so as not to have to update this module every time.